### PR TITLE
ani-cli: 2.2 -> 3.2

### PR DESCRIPTION
**Pull request by Creator54**
###### Description of changes
https://github.com/pystardust/ani-cli/releases/tag/v3.2

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested on:
  - [ NixOS ]
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by skykanin — APPROVED** [review]
LGTM

**Review by skykanin — APPROVED** [review]

File: pkgs/applications/video/ani-cli/default.nix [hunk 1]
```diff
@@ -12,26 +14,29 @@
...
+    install -Dm755 ./bin/ani-cli $out/bin/ani-cli
+
+    mv ./lib $out/
 
     wrapProgram $out/bin/ani-cli \
-      --prefix PATH : ${lib.makeBinPath [ gnugrep gnused curl openssl mpv aria2 ]}
+      --prefix PATH : ${lib.makeBinPath [ gnugrep gnused curl openssl mpv aria2 gawk ffmpeg ]}
 
     runHook postInstall
+
```
Comment (SuperSandro2000) [inline pkgs/applications/video/ani-cli/default.nix:39]:
```suggestion
```

File: pkgs/applications/video/ani-cli/default.nix [hunk 2]
```diff
@@ -12,24 +14,26 @@
...
   pname = "ani-cli";
-  version = "2.2";
+  version = "3.2";
 
   src = fetchFromGitHub {
     owner = "pystardust";
     repo = "ani-cli";
     rev = "v${version}";
-    sha256 = "sha256-B/bIGrSQmKZFh3PpsbwivR2QKLMHIypWLxWuufiFHw4=";
+    sha256 = "LSyqcJX+bPW/qtO3FaZh9RaXW2xeGc/9tdC2CmhDwUw=";
```
Comment (SuperSandro2000) [inline pkgs/applications/video/ani-cli/default.nix:23]:
```suggestion
    sha256 = "sha256-LSyqcJX+bPW/qtO3FaZh9RaXW2xeGc/9tdC2CmhDwUw=";
```

**Comment by SuperSandro2000** [comment]
Please also rebase on master to resolve the merge conflict after #183135

**Comment by Creator54** [comment]
> Please also rebase on master to resolve the merge conflict after #183135

@SuperSandro2000 
should I change the git commit message too ??If yes what to ?  because ani-cli is already been updated to 3.3 , which is broken and my commit will just fix it not actually update the package.

**Comment by Creator54** [comment]
Oops just now rechecked ani-cli is working as expected, so need for this.

**Pull request closed without merging**